docs/content/Configuration/Environment-Variables-Reference.md

```diff
@@ -115,6 +115,12 @@ databases [in this guide][link-connecting-to-db].
 | `CUBEJS_DB_SNOWFLAKE_ROLE`                      | Snowflake            | The Snowflake role to use when connecting to the database                                                                                                                                                   | A valid Snowflake role for the account                           |
 | `CUBEJS_DB_SNOWFLAKE_WAREHOUSE`                 | Snowflake            | The Snowflake warehouse to use when connecting to the database                                                                                                                                              | A valid Snowflake warehouse for the account                      |
 | `CUBEJS_DB_SNOWFLAKE_CLIENT_SESSION_KEEP_ALIVE` | Snowflake            | If `true`, (keep the Snowflake connection alive indefinitely)[link-snowflake-connection-options]                                                                                                            | `true`, `false`                                                  |
+| `CUBEJS_DB_SNOWFLAKE_AUTHENTICATOR`                       | Snowflake            | The type of authenticator to use with Snowflake. <br/> `Username + Password`(`SNOWFLAKE`)<br/> `Key-Pair`(`SNOWFLAKE_JWT`)<br/> If this variable is not provided, default to `SNOWFLAKE` | `SNOWFLAKE`, `SNOWFLAKE_JWT`  |
```

Comment:
The table seems to be broken on the Netlify Deploy Preview: https://deploy-preview-2724--cubejs-docs-staging.netlify.app/reference/environment-variables#database-connection

Reply:
@hassankhan 
Done ✅